packages/ember-toucan-core/src/components/form/fields/input.ts

```diff
@@ -62,4 +67,12 @@ export default class ToucanFormInputFieldComponent extends Component<ToucanFormI
   get hasError() {
     return Boolean(this.args?.error);
   }
+
+  /**
+   * Used to help us determine if we should render a single error
+   * or render a ul list of errors.
+   */
+  get hasMoreThanOneError() {
+    return Array.isArray(this.args.error) && this.args.error?.length > 1;
+  }
```

Comment:
Keeping `error.hbs` as is as suggested at https://github.com/CrowdStrike/ember-toucan-core/pull/115/files#diff-cfd3d39e514acb00199182387cf476ea65b2cb7c105843faeac73c111413df8d means this could likely go away too!

Reply:
This looks like it's still needed with the `if (has-block)` condition we added.